python/gtsam/preamble.h

```diff
@@ -6,9 +6,17 @@ PYBIND11_MAKE_OPAQUE(std::vector<gtsam::Key, tbb::tbb_allocator<gtsam::Key>>);
 PYBIND11_MAKE_OPAQUE(std::vector<gtsam::Key>);
 #endif
 PYBIND11_MAKE_OPAQUE(std::vector<gtsam::Point2, Eigen::aligned_allocator<gtsam::Point2> >);
+PYBIND11_MAKE_OPAQUE(gtsam::Point3Pairs);
```

Comment:
thanks for the PR, Fan. Was this opaque option something we forgot to add earlier? What's the advantage of it -- preventing a deep copy?

Reply:
Yes, it prevents a deep copy.

Reply:
This tells Pybind11 not to perform a cpp-to-python conversion of types and instead treat this as its own type. This help prevents issues with pass-by-reference propagation between the two languages.

Reply:
^ much better explanation

Reply:
Make sure this much better explanation is copied to wrapper docs.

Reply:
done